### PR TITLE
Adding ADKami basic interactions

### PR DESCRIPTION
**Pull request by antharuu**
I just added a simple interaction for ADKami.

You can see what you are browsing as well as what you are looking at.

Being a French site with a French content, I put the texts only in French.

adkami-premid-3

adkami-premid-1

adkami-premid-2

### CODE REVIEW

File: websites/A/ADKami/presence.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+       descr = descrParts.join(" ");
+
+      presenceData.details = titleParts.join("-");
+      presenceData.state = `${descr} (${lang})`;
+
+      const bigImage: string = document.querySelector("#body > section > div.col-12.col-l-9.fiche-look > div.row.blocshadow > div > img.col-12.m-hidden").getAttribute("src");
+
+      presenceData.smallImageText = bigImage;
+
+      console.log(bigImage);
```
Comment (EncryptedDev) [inline websites/A/ADKami/presence.ts:29]: Why do you have this?

File: websites/A/ADKami/presence.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  const presenceData: PresenceData = {
+    largeImageKey: "logo"
+  },
+
+   path: string = document.location.pathname;
+  let cat: string = path.split("/")[1];
+
+  if (cat === "drama-streaming") cat = "drama";
+
+  if (cat !== "") {
```
Comment (EncryptedDev) [inline websites/A/ADKami/presence.ts:13]: Checking if it is an empty string?

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Hi, not much to this presence, quite basic indeed - support more endpoints?
Look at my review too

File: websites/A/ADKami/presence.ts (hunk 1 above)
Comment (antharuu) [reply websites/A/ADKami/presence.ts:29]: Oh, indeed I forgot to remove this, it was for testing.

File: websites/A/ADKami/presence.ts (hunk 2 above)
Comment (antharuu) [reply websites/A/ADKami/presence.ts:13]: Yes, the string is empty if you are on the home page.

File: websites/A/ADKami/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  },
+  "service": "ADKami",
+  "description": {
+    "fr": "Streaming d'Anime, Manga, Drama, Séries, Hentai",
+    "en": "Anime, Manga, Drama, Séries, Hentai Streaming"
+  },
+  "url": "adkami.com",
+  "version": "1.0.0",
+  "logo": "https://www.adkami.com/favicon2.png",
+  "thumbnail": "https://streaming-one.com/wp-content/uploads/2021/03/FireShot-Capture-1547-Accueil-ADKami-m.adkami.com_-1024x496.png",
```
Comment (EncryptedDev) [inline websites/A/ADKami/dist/metadata.json:14]: Also, these both need to be on imgur

Comment (antharuu) [reply websites/A/ADKami/dist/metadata.json:14]: Ah I didn't know. I'll update it soon then.

**Comment by ririxidev** [comment]
I'm sorry but I need to decline this presence;
- presence doesn't have a lot to offer
- mistakes within JS (no consts, messing up with variables)
- no imgur links
- weird variable naming
- and more...

**Comment by antharuu** [comment]
> * presence doesn't have a lot to offer

That's pretty subjective... what can you base that on? Would the code be too short and simple for you?

It just brings what a basic integration brings. What do you expect from a streaming site like this?

Be constructive at least on this point.

**Comment by ririxidev** [comment]
No support for media player, no support for all subpages etc 😃

**Comment by antharuu** [comment]
Well, the subpages are useless because the title and the number of the episode is included in the page name. What do you expect from this ?

As for the media player, it seems complicated to me for a site like this one which includes dozens (or more) media players from third party sites. Especially since it would not bring much...

**Comment by ririxidev** [comment]
Well sorry then, no these features - no presence in store :/

**Comment by antharuu** [comment]
It seems unjustified to me, but so be it. I'll do what you want if I have the time...

**Pull request closed without merging**